### ISSUE TITLE
Can't install…

### ISSUE THREAD

**Issue by utilisezlinux** [ucsf-ckm/symfony-emberjs-edition#7]
dependency problem with "sensio/generator-bundle": "~2.3"

**Comment by Soullivaneuh** [comment]
Hi @utilisezlinux, I fixed it by using `~3.0` version.

Here, all my requirements:

```json
    "require": {
        "php": ">=5.3.3",
        "symfony/symfony": "~2.3",
        "doctrine/orm": "~2.4",
        "doctrine/doctrine-bundle": "~1.2",
        "twig/extensions": "~1.0",
        "symfony/swiftmailer-bundle": "~2.3",
        "symfony/monolog-bundle": "~2.3",
        "sensio/distribution-bundle": "~3.0",
        "sensio/framework-extra-bundle": "~3.0",
        "sensio/generator-bundle": "~2.3",
        "incenteev/composer-parameter-handler": "~2.0",
        "kriswallsmith/assetic": "dev-master as 1.2",
        "symfony/assetic-bundle": "dev-master",
        "friendsofsymfony/rest-bundle": "1.4.*",
        "jms/serializer-bundle": "0.13.*",
        "nelmio/api-doc-bundle": "2.*",
        "friendsofsymfony/jsrouting-bundle": "~1.5",
        "sp/bower-bundle": "dev-master"
    },
```

**Comment by utilisezlinux** [comment]
Ok thanks
